battleship-server/src/main/java/com/epam/solid/nie/config/language/LanguageConfigurationImpl.java

```diff
@@ -10,7 +10,7 @@
 
     @Override
     public Language provide() {
-        ResourceBundle resourceBundle = ResourceBundle.getBundle(Configuration.CONFIGURATION);
+        ResourceBundle resourceBundle = ResourceBundle.getBundle(FileConfiguration.CONFIGURATION);
```

Comment:
Oh I see. Then, let's change the CONFIGURATION to NAME. That way it's `FileConfiguration.NAME` and it becomes more readable.